File: c_language/Sejong/일반_C/인증/212차모의인증/1.c
#include <stdio.h>

int main(){
    int x[3], num, count, i,j, tmp;
    while(1){
        count = 0;
        for(i = 0; i < 3; i++){
            scanf("%d", &x[i]);
            if(x[i] == 0){
                count++;
            }
        }
        if(count == 3){
            break;
        }
        for(i = 0; i < 2; i++){
            for(j = i + 1; j < 3; j++){
                if(x[j] < x[i]){
                    tmp = x[i];
                    x[i] = x[j];
                    x[j] = tmp;
                }
            }
        }
        for(i = 2; i >= 0; i--){
            printf("%d ", x[i]);
        }
        printf("\n");
    }


    return 0;
}

File: baekjoon/math_2/1.py
# 1978
import sys, math

def check_prime(x):
	if x == 1:
		return 0
	for i in range(2, int(math.sqrt(x)) + 1):
		if x % i == 0:
			return 0
	return 1

n = int(sys.stdin.readline())
count = 0
num = list(map(int, sys.stdin.readline().split()))
for i in num:
	if check_prime(i):
		count += 1
	
print(count)
	
